.github/workflows/typescript_lint.yaml

```diff
@@ -0,0 +1,53 @@
+name: PR Workflow
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      - "website/**"
+
+jobs:
+  linting_and_type-checking:
+    name: Linting, Formatting and Type checking
```

Comment:
The job name as well, we should mention it is related to react/ts

Reply:
Done this one also : )